### ISSUE TITLE
Could be clearer that Cross Entropy takes logits as input

### ISSUE THREAD

**Issue by Alex7Li** [pytorch/pytorch#82081]
### 📚 The doc issue

Hi, I felt that it could be made a bit more clear that the cross entropy function is taking logits in as input. Right now, the documentation is correct but I felt that it could be a bit more clear. It says that the input is required to take "raw, unnormalized scores for each class", but for me when reading that, I was a bit uncertain about what that meant. Then I scrolled down a bit more and saw that (target) would take in probabilities as input and incorrectly started to believe that cross entropy takes probabilities as input. Especially since that is the normal mathematical definition of cross entropy, I think it would be nice to be more clear; a lot of people will assume that it takes probabilities and get confused; I noticed that this question has a lot of upvotes 
https://stackoverflow.com/questions/49390842/cross-entropy-in-pytorch  on stackoverflow

### Suggest a potential alternative/fix

Instead of 
`The input is expected to contain raw, unnormalized scores for each class.`
it could say
`The input is expected to contain the logits for each class (which do not need to be positive or sum to 1, in general). This class will compute a softmax followed by cross entropy to compute the final loss.`

Of course, if one reads the documentation carefully enough (e.g. the formulas) they will eventually figure it out, but if you come in with bad assumptions like me and don't read carefully then you might spend a really long time trying to debug your model! Since it's a popular page I think it's worth it to be super clear

**Comment by jbschlosser** [comment]
Hey @Alex7Li, thanks for the report! Note that the functional [`F.cross_entropy()` docs](https://pytorch.org/docs/stable/generated/torch.nn.functional.cross_entropy.html) do include a statement referring to expected input as logits. We'd accept a PR clarifying this in the `nn.CrossEntropyLoss` module docs as well if you'd like to contribute!
